interface/modules/custom_modules/oe-module-comlink-telehealth/sql/table.sql

```diff
@@ -0,0 +1,88 @@
+#IfNotTable comlink_telehealth_auth
+CREATE TABLE IF NOT EXISTS `comlink_telehealth_auth`
+(
+    `id`              	      BIGINT(20)         NOT NULL AUTO_INCREMENT,
+    `username`                VARCHAR(64) NOT NULL COMMENT 'Username used for external access',
+    `user_id`                 BIGINT(20)         NULL COMMENT 'Foreign key reference to users.id',
+    `patient_id`              BIGINT(20)         NULL COMMENT 'Foreign key reference to patient_data.id',
+    `auth_token` 	          TEXT NOT NULL COMMENT 'external authorization token to use telehealth api',
```

Comment:
remove the NOT NULL (we keep all TEXT family fields bare)

Reply:
Removed.